File: core/src/net/bossdragon/enums/CollisionGroups.java
package net.bossdragon.enums;

public interface CollisionGroups {
    long CHARACTER = 1;
    long SOLDIER = 1 << 1;
    long BULLET = 1 << 2;
    long WALL = 1 << 3;
    long DRAGON = 1 << 4;
    long FIREBALL = 1 << 5;
    long BURNING_AREA = 1 << 6;
}

File: core/src/net/bossdragon/enums/CollisionGroups.kt
package net.bossdragon.enums

object CollisionGroups {
    val CHARACTER: Long = 1
    val SOLDIER = (1 shl 1).toLong()
    val BULLET = (1 shl 2).toLong() //made of enemy
    val WALL = (1 shl 3).toLong()
    val DRAGON = (1 shl 4).toLong()
    val FIREBALL = (1 shl 5).toLong()
    val BURNING_AREA = (1 shl 6).toLong()
}
